source/whats-new.txt

```diff
@@ -41,8 +41,8 @@ New features of the 4.6 Java driver release include:
   a custom DNS client. See the `DnsClient <https://mongodb.github.io/mongo-java-driver/4.6/apidocs/mongodb-driver-core/com/mongodb/spi/dns/DnsClient.html>`__
   and the `DnsClientProvider <https://mongodb.github.io/mongo-java-driver/4.6/apidocs/mongodb-driver-core/com/mongodb/spi/dns/DnsClientProvider.html>`__
   interface API documentation for more information.
-- Added the `` org.mongodb:bson-record-codec`` library for encoding and
-  decoding Java records using the ``RecordCodec`` and ``RecordCodecProvider``.
+- Added driver support for encoding and decoding between Java Records and
```

Comment:
Nit: lowercase `Records`.
Nit: Add a link to https://docs.oracle.com/en/java/javase/17/language/records.html or https://docs.oracle.com/javase/specs/jls/se18/html/jls-8.html#jls-8.10